tests/gis_tests/geoapp/tests.py

```diff
@@ -63,9 +63,11 @@ def test_proxy(self):
         nullcity.point.x = 23
         nullcity.point.y = 5
         # Checking assignments pre & post-save.
-        self.assertNotEqual(Point(23, 5), City.objects.get(name='NullCity').point)
+        self.assertNotEqual(Point(23, 5, srid=4326),
```

Comment:
Longer lines are fine here (we allow up to 119 characters when it improves readability).